bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -144,34 +135,34 @@ class Base extends Component {
   renderByState() {
     const { updateLoadingState } = this;
     const stateControls = { updateLoadingState };
-
-    const { ejected } = this.props;
-
-    const { loading, meetingExisted } = this.state;
-
+    const { loading, isBreakout } = this.state;
     const codeError = Session.get('codeError');
     const {
+      ejected,
       subscriptionsReady,
-      meetingExist,
+      meetingIsEnded,
     } = this.props;
+    const isMeetingBreakout = AppService.meetingIsBreakout();
 
     if (ejected && ejected.ejectedReason) {
       const { ejectedReason } = ejected;
       AudioManager.exitAudio();
       return (<MeetingEnded code={ejectedReason} />);
     }
 
-    if ((meetingExisted && !meetingExist)) {
+    if (isMeetingBreakout && !isBreakout) this.setState({ isBreakout: true });
+
+    if (meetingIsEnded && !isBreakout) {
       AudioManager.exitAudio();
-      return (<MeetingEnded code={Session.get('codeError')} />);
+      return (<MeetingEnded code={codeError} />);
```

Comment:
In MeetingEnded Component the codeError is required
But, The Base Component not define a default value in case of the codeError not be able to found in Session
Should we set a default value for codeError?